ui/src/widgets/ui-table/UITable.vue

```diff
@@ -128,14 +128,20 @@ export default {
             }
         },
         onRowClick (row) {
-            console.log(row)
+            if (this.selected === null) {
+                this.selected = row;
+            } else if (this.selected === row) {
+                this.selected = null;
+            } else {
+                this.selected = row;
+            }
```

Comment:
I think this can be achieved with one conditional statement? But my logic review might be incorrect?
```suggestion
            this.selected = this.selected === row ? null : row;
```